app/views/shared/_header.html.slim

```diff
@@ -0,0 +1,20 @@
+a(class="navbar-brand" href="/")
+  | Home
+
+#navbarSupportedContent.collapse.navbar-collapse
+  ul.navbar-nav.mr-auto
+    li.nav-item
+      a(class="nav-link" href="/trucks")
+        | Trucks
+    li.nav-item
+      a(class="nav-link" href="/drivers")
+        | Drivers
+    li.nav-item
+      a(class="nav-link" href="/routes")
+        | Routes
+    li.nav-item
+      a(class="nav-link" href="/cargos")
+        | Cargo
+    li.nav-item
+      a(class="nav-link" href="/model_types")
+        | Model Type
```

Comment:
We have the potential to dry this up if we want to get tricky.

```suggestion
    - ['trucks', 'drivers', 'routes', 'cargos', 'model_types'].each do |item|
      li.nav-item
        a(class="nav-link" href="/#{item}"
          = item.titleize
```

Reply:
![image](https://user-images.githubusercontent.com/24546647/99241189-7b7cfb00-2838-11eb-86a2-e182ef707062.png)
I receive an error

Reply:
` a(class="nav-link" href="/#{item}")`
was missing an ending `)`